eng/pipelines/templates/jobs/installation.check.yml

```diff
@@ -0,0 +1,33 @@
+parameters:
+  - name: Artifact
+    type: object
+    default: {}
+  - name: ArtifactName
+    type: string
+    default: "not-specified"
+
+jobs: 
+  - job: InstallationCheck
+    displayName: "Installation Check"
+    variables:
+      - template: ../variables/globals.yml
+    steps:
+      - download: current
+        artifact: ${{parameters.ArtifactName}}-signed
+      - pwsh: |
+          Write-Host "Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}-signed/${{parameters.Artifact.groupId}}/${{parameters.Artifact.name}}/" "$(Pipeline.Workspace)/install-check-dir/${{parameters.Artifact.name}} -Recurse"
+          Copy-Item "${{parameters.ArtifactName}}-signed/${{parameters.Artifact.groupId}}/${{parameters.Artifact.name}}/" "install-check-dir/${{parameters.Artifact.name}}" -Recurse
+          Write-Host "$pomf = Get-ChildItem install-check-dir/${{parameters.Artifact.name}}/*.pom"
+          $pomf = Get-ChildItem install-check-dir/${{parameters.Artifact.name}}/*.pom
+          Write-Host "Rename-Item $($pomf) pom.xml"
+          Rename-Item $pomf pom.xml
+          Copy-Item "$(System.DefaultWorkingDirectory)/eng" "install-check-dir/${{ parameters.Artifact.name }}/eng" -Recurse
+        workingDirectory: $(Pipeline.Workspace)
+        displayName: "Copy Artifacts and Directories"
+      - template: /eng/pipelines/templates/steps/install-reporting-tools.yml
+      - task: Maven@3
+        displayName: "Verify Package Installation"
```

Comment:
We might need to put this in a script so that we can do a retry similar to what we are doing for .NET

Reply:
The createReleaseTag + publish to maven jobs normally finishes within 15 minutes. I'd say make it 30 minutes just to be sure.